src/common/fmt_common.h

```diff
@@ -17,8 +19,13 @@
  * such classes in Crimson.
  */
 
-template <typename T>
-concept has_formatter = fmt::has_formatter<T, fmt::format_context>::value;
+#if FMT_VERSION < 110000
```

Comment:
see https://github.com/fmtlib/fmt/commit/1752d7fbbbeb5ae2fb30bad5f370a03c3f82e7c7 for the change which introduced this concept to fmtlib.